docs/developer-docs/latest/setup-deployment-guides/configurations/optional/functions.md

```diff
@@ -26,6 +27,7 @@ It can be used to:
 - create an admin user if there isn't one
 - fill the database with some necessary data
 - declare custom conditions for the [Role-Based Access Control (RBAC)](/developer-docs/latest/setup-deployment-guides/configurations/optional/rbac.md) feature
+- register the admin panel part of [custom fields](/developer-docs/latest/development/custom-fields.md) that would be used only by the current Strapi application.
```

Comment:
this is done in the register lifecycle not bootstrap

Reply:
So this is different from what was [mentioned in the RFC](https://github.com/strapi/rfcs/blob/3eb034e169746558315d719ca5fb49cec854640a/rfcs/xxxx-custom-fields-api.md#admin)? 🤔

Reply:
good catch, there does seem to be a discrepancy between that and what is in [the example](https://github.com/strapi/rfcs/blob/3eb034e169746558315d719ca5fb49cec854640a/rfcs/xxxx-custom-fields-api.md#example) which uses the `register` lifecycle. 

It was always my understanding that the custom field is registered in the `register` lifecycle for the admin and server for both plugin and Strapi applications. I want to say `bootstrap` mentioned in the RFC was typo. @remidej could you confirm?